pipo/settings.yaml

```diff
@@ -47,3 +47,8 @@ default:
       pool_size: 2
       retries: 3
       wait: 1
+    source:
+      youtube:
+      spotify:
+        playlist_limit: 100
+        album_limit: 100
```

Comment:
Why are these limits necessary? Where are they being used?

Reply:
Used in spotify_handler to avoid a big playlist or album to DoS the system.